### DISCUSSION TITLE
Any possibility to change [[[ ]]] syntax, to make it comfortable work in external editors.

### DISCUSSION THREAD

**Discussion by GenrihGrigoryan** [srid/neuron#494] [Ideas]
As idea - choosing special symbol in neuron.dhall: “heterarchy_symbol: ^ ”

After that, i can use [[link]]^ as replacing for [[[link]]], to work with comfort in external markdown editors (now, triple brackets doesn’t reckognising by any editor (obsidian, zettlr, vscode) and disturb from normal workflow - links doesn’t creating automaticly, visual graph doesn’t see folgezettel links, etc).

**Comment by srid** [comment]
Not a bad idea, actually.

Note that [vscode-memo](https://github.com/svsool/vscode-memo) does recognize `[[[...]]]`.
